docs/prysm-usage/get-more-peers.md

```diff
@@ -0,0 +1,59 @@
+---
+id: get-more-peers
+title: Get More Peers
+sidebar_label: Get more peers
+---
+
+ETH2 is designed to be a fully peer to peer (P2P) network.  This section describes how to configure Prysm and your network to optimise the number of peers that you communicate with on the ETH2 Network.  
```

Comment:
Good moment to link back to the on ethereum 2.0 and architecture overview pages
2) may seem a bit redundant, but would be good to note in this paragraph that increasing peers both helps the users connectivity as well as the overall network

Reply:
done